apps/pie-docs/src/components/components.json

```diff
@@ -0,0 +1,4 @@
+{
+  "metaTitlePrefix": "Components — ",
```

Comment:
Not sure where this is used, but usually I wouldn't recommend making separators part of the individual parts.

Have you considered something like `[...parts].join(' - ')`?

Reply:
Hey, I'll get this in as a separate ticket so all of them are done in one go :)

Reply:
@webbertakken I'll send it to you on slack